modules/internal/ArrayViewReindex.chpl

```diff
@@ -109,7 +109,7 @@ module ArrayViewReindex {
       }
     }
 
-    iter these(param tag: iterKind) ref where tag == iterKind.standalone {
+    iter these(param tag: iterKind) ref where tag == iterKind.standalone && !localeModelHasSublocales {
```

Comment:
Yes, we should fall back on the leader-follower iterators. I'd rather not have to make this change, but otherwise we run into a variety of issues with reductions over arrays.

I believe there is some precedent for param-choosing iterators in DefaultRectangular based on the locale-model.